### PR TITLE
elasticsearch cluster for windows integrated with Linux template

### PR DESCRIPTION
**Pull request by kirpasingh**
Modified the deployment template to include Windows extension settings, command line to run powershell script on each node and added a powershell script that runs as a custom extension and sets up elasticsearch cluster on Azure for Windows.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kirpasingh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kirpas). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: elasticsearch/azuredeploy.json [hunk 1]
```diff
@@ -160,7 +161,7 @@
     }
   },
   "variables": {
-    "templateBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch/",
+    "templateBaseUrl": "https://raw.githubusercontent.com/kirpasingh/azure-quickstart-templates/master/elasticsearch/",
```
Comment (singhkay) [inline elasticsearch/azuredeploy.json]: Please make sure to change this back to Azure location when you're done testing

File: elasticsearch/azuredeploy.parameters.json [hunk 2]
```diff
@@ -3,13 +3,13 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "dnsNameforLBIP": {
-      "value": "changeme"
+      "value": "elasticlbip"
```
Comment (singhkay) [inline elasticsearch/azuredeploy.parameters.json]: Please change this value to "#####". Helps the automated bot deploy this and replace this with a random gen'd string

File: elasticsearch/azuredeploy.parameters.json [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
     },
     "storageAccountPrefix": {
-      "value": "changeme"
+      "value": "elasticstore"
```
Comment (singhkay) [inline elasticsearch/azuredeploy.parameters.json]: Please change this value to "#####". Helps the automated bot deploy this and replace this with a random gen'd string

File: elasticsearch/azuredeploy.parameters.json [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
     },
     "adminUsername": {
-      "value": "trent"
+      "value": "admin"
```
Comment (singhkay) [inline elasticsearch/azuredeploy.parameters.json]: This username might be blocked on the platform, will cause the automated bot deployment to fail

File: elasticsearch/azuredeploy.parameters.json [hunk 5]
```diff
@@ -3,13 +3,13 @@
...
     },
     "adminPassword": {
       "value": "changeme"
```
Comment (singhkay) [inline elasticsearch/azuredeploy.parameters.json]: Can you change this value to something that passes password requirements, something like Chang3m3 maybe?

**Pull request merged**